.gitlab-ci.yml

```diff
@@ -31,13 +31,15 @@ test:unit:
 
 test:govendor-check:
   variables:
-    GOLANG_VERSION: "1.18"
+    GOLANG_VERSION: '1.18'
 
-test:acceptance:
+test:acceptance_tests:
   stage: test
   needs: []
-  except:
-    - /^saas-[a-zA-Z0-9.-]+$/
+  rules:
+    - if: '$CI_COMMIT_TAG =~ /^saas-[a-zA-Z0-9.]+$/'
+      when: never
```

Comment:
every repo I checked has the except, why are we moving to if? if we do, we need a task to change in all the repos, otherwise there are going to be questions "which standard to use" and further mess.

Reply:
both are supported, the rules one is the one to align to (https://docs.gitlab.com/ee/ci/yaml/index.html#only--except) - they are migrated whenever those things are touched, but since there is no plan for a deprecation there is no hurry and no task for this yet.

Reply:
also https://github.com/mendersoftware/mender-test-containers/pull/342#discussion_r1230839943 as motivation